dnf/cli/commands/repoquery.py

```diff
@@ -16,22 +16,23 @@
 # Red Hat, Inc.
 #
 
-from __future__ import absolute_import
-from __future__ import print_function
-from __future__ import unicode_literals
-from datetime import datetime
-from dnf.i18n import _
-from dnf.cli import commands
+from __future__ import (absolute_import, print_function, unicode_literals)
 
 import argparse
+import logging
+import re
+import sys
```

Comment:
A am sorry, but the present order is used in whole dnf, therefore I would not recommend to change it.